src/app/item-popup/ItemStats.tsx

```diff
@@ -64,15 +64,20 @@ function ItemStatRow({
 }) {
   const value = stat.value || 0;
   const compareStatValue = (compareStat ? compareStat.value : 0) || 0;
+  // lower # is better for drawtime and chargetime stats
+  const lowerBetter = [447667954, 2961396640].includes(stat.statHash);
 
   const isMasterworkedStat =
     item.isDestiny2() && item.masterworkInfo && stat.statHash === item.masterworkInfo.statHash;
   const masterworkValue =
     (item.isDestiny2() && item.masterworkInfo && item.masterworkInfo.statValue) || 0;
-
   const higherLowerClasses = {
-    'higher-stats': value > compareStatValue && compareStat,
-    'lower-stats': value < compareStatValue && compareStat
+    'higher-stats':
+      ((!lowerBetter && value > compareStatValue) || (lowerBetter && value < compareStatValue)) &&
```

Comment:
'higher-stats': !lowerBetter ? value > compareStatValue && compareStat : value < compareStatValue && compareStat;

Reply:
as you wish~